wagtail/admin/templates/wagtailadmin/shared/side_panels/includes/status/privacy.html

```diff
@@ -2,7 +2,7 @@
 {% load i18n wagtailadmin_tags %}
 
 {% block content %}
-    {% test_page_is_public page as is_public %}
+    {% if page.id %} {% test_page_is_public page as is_public %} {% else %} {% test_page_is_public parent_page as is_public %} {% endif %}
```

Comment:
```suggestion
    {% if page.id %}{% test_page_is_public page as is_public %}{% else % {% test_page_is_public parent_page as is_public %}{% endif %}
```

Just a small nitpick, no need to add spaces between the template tags here.

Reply:
Something like what's done here. 

https://github.com/wagtail/wagtail/blob/cc335cad5f88e888b0cc7dd89e3352cc68ea2f94/wagtail/admin/forms/pages.py#L13-L17